tests/SalesFileUploadCommandTest.php

```diff
@@ -159,7 +159,7 @@ function ($notification, $channels, $notifiable) {
         );
     });
 
-    it('uploads sales files to SFTP server with enabled failure notifications only', function () {
+    it('uploads sales files to SFTP server when only failure notifications are enabled', function () {
```

Comment:
No Harsh. The upload happens in any case. Only the notification is not sent.
You did properly here - https://github.com/Retail-Cosmos/ioi-city-mall-sales-file/pull/38/files#diff-bb148c5c74762144ee11f44a280aa90775d26c2a3014ef0bf9e23d643229a6dfR423

What happened?